documentation/Demo-Server-Guide.md

```diff
@@ -121,7 +159,9 @@ Logs are in `/var/log/nginx`
 
 ## docker-compose
 
-docker-compose will complain if directories are not present, create them if needed (they will persist when restarting image, even if `docker-compose down` was run)
+docker-compose will complain if directories are not present, create them if needed (they will persist when restarting image, even if `docker-compose down` was run) - use `docker compose down` alternativelly.
```

Comment:
This is actually not true, but it doesn't matter for now cos I'll be changing it in my next PR for #710